python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -383,7 +411,19 @@ def emit(self, record: logging.LogRecord) -> None:
 class PipesContextLoader(ABC):
     @abstractmethod
     @contextmanager
-    def load_context(self, params: PipesParams) -> Iterator[PipesContextData]: ...
+    def load_context(self, params: PipesParams) -> Iterator[PipesContextData]:
+        """A `@contextmanager` that loads context data injected by the orchestration process.
```

Comment:
should we be more explicit about what the "context" is?

Reply:
Discussed with @schrockn in another PR, concluded that we should keep the docstrings concise and leave the detailed explanation of this stuff to prose docs. Otherwise every `load_context` implementation etc would have a huge repetitive docstring.